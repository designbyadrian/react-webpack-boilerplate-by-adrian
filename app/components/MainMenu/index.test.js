import React from 'react';
import { render } from 'react-testing-library';

import Menu from './index';

describe('components/Menu', () => {
  let store;

  beforeEach(() => {
    store = { routing: {} };
  });

  it('mounts', () => {
    const { container } = render(<Menu routing={store.routing} />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
