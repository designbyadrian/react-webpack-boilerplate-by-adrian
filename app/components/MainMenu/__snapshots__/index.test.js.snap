// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Menu mounts 1`] = `
<nav
  class="navbar navbar-expand-lg navbar-light bg-light"
>
  <div
    class="container"
  >
    <a
      href="/"
    >
      React 
      <strong>
        16
      </strong>
    </a>
    <ul
      class="navbar-nav mr-auto"
    >
      <li
        class="nav-item "
      >
        <a
          href="/"
        >
          menu.home
        </a>
      </li>
      <li
        class="nav-item "
      >
        <a
          href="/settings"
        >
          menu.settings
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
